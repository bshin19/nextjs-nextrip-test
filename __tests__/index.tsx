import { render } from "@testing-library/react";
import Home from "../pages";

describe("Home", () => {
  it("should match default", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
