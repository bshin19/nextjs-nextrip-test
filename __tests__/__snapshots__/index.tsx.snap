// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match default 1`] = `<div />`;
